src/components/va-select/va-select.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  State,
+  h,
+} from '@stencil/core';
+import { getSlottedNodes } from '../../utils/utils';
+
+@Component({
+  tag: 'va-select',
+  styleUrl: 'va-select.css',
+  shadow: true,
+})
+export class VaSelect {
+  @Element() el: any;
+
+  /**
+   * Whether or not this is a required field.
+   */
+  @Prop() required: boolean;
+
+  /**
+   * Text label for the field.
+   */
+  @Prop() label: string;
+
+  /**
+   * Name attribute for the select field.
+   */
+  @Prop() name: string;
+
+  /**
+   * Selected value (will get updated on select).
+   */
+  @Prop({ reflect: true, mutable: true }) value: string;
+
+  /**
+   * Error message to display. When defined, this indicates an error.
+   */
+  @Prop() error: string;
+
+  /**
+   * Optional text that is displayed to screen readers only.
+   */
+  @Prop() ariaLiveRegionText: string;
```

Comment:
Unless there's a back support need for this prop, I'd ask if we can remove it.

Reply:
This is great news, I love simplifying things! It looks like it isn't used in any apps, so I'll go ahead and remove it.

![image](https://user-images.githubusercontent.com/2008881/121727808-3e60aa80-caa1-11eb-840b-d6985e364b53.png)